Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -212,10 +221,22 @@ public final class EpoxySwiftUIHostingView<RootView: View>: UIView, EpoxyableVie
         trailing: margins.right)
     }
 
-    // Allow the layout margins update to fully propagate through to the SwiftUI View before
-    // invalidating the layout.
-    DispatchQueue.main.async {
-      self.viewController.view.invalidateIntrinsicContentSize()
+
+    if forceLayoutOnLayoutMarginsChange {
+      // If we don't force a layout pass and size invalidation synchronously after the layout
```

Comment:
add documentation that we'll consider making this the only behavior after testing for a bit?